cmd/dp-dimension-importer/main.go

```diff
@@ -25,29 +41,36 @@ func (r responseBodyReader) Read(reader io.Reader) ([]byte, error) {
 
 func main() {
 	log.Namespace = "dimension-importer"
+
+	errorsChan := make(chan error)
```

Comment:
Not currently using this anywhere - I am guessing you will pass this into objects to pass errors back up?

Reply:
Out dated, removed.